Quadriga/src/main/webapp/WEB-INF/views/navigation/nav-workspace.jsp

```diff
@@ -74,4 +72,22 @@
 		</li>
 		 -->
 	</ul>
-</div>
\ No newline at end of file
+</div>
+<h5 class="major" style="margin-top: 0.5em;margin-bottom: 0em;">
+	<span>DICTIONARIES</span>
+</h5>
+<div id="projectDictionaries">
+	<img src="/quadriga/resources/txt-layout/css/images/plus.png" style="vertical-align: middle; padding-bottom: 2px;">
+	<a href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/adddictionary">Add</a>
+	<img src="/quadriga/resources/txt-layout/css/images/minus.png" style="vertical-align: middle; padding-bottom: 2px;">
+	<a href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/deletedictionary">Delete</a>
+</div>
+<h5 class="major" style="margin-top: 0.5em;margin-bottom: 0em;">
+	<span>CONCEPT COLLECTIONS</span>
+</h5>
+<div id="projectConceptCollections">		
+	<img src="/quadriga/resources/txt-layout/css/images/plus.png" style="vertical-align: middle; padding-bottom: 2px;">
+	<a href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/addconceptcollection">Add</a>
+	<img src="/quadriga/resources/txt-layout/css/images/minus.png" style="vertical-align: middle; padding-bottom: 2px;">
+	<a href="${pageContext.servletContext.contextPath}/auth/workbench/${project.projectId}/deleteconceptcollections">Delete</a>

```

Comment:
please merge develop in this branch and use font awesome icons instead of the images